src/v2/Apps/Conversation/Components/Reply.tsx

```diff
@@ -67,7 +68,7 @@ export const Reply: React.FC<ReplyProps> = props => {
           textArea.current.style.height = "inherit"
           setLoading(false)
           setButtonDisabled(true)
-
+          onScroll()
```

Comment:
@lilyfromseattle - if `onScroll` here is `null` (the default argument) this will result in a runtime error.

Reply:
Ok, how should I avoid that?

Reply:
Just check to ensure it's non-null before calling it

Reply:
A simple guard would suffice:
```js
if (onScroll) {
  onScroll() 
}
```